utils/chain-ops/genesis/config.go

```diff
@@ -224,8 +223,17 @@ func (d *DeployConfig) Check() error {
 	if d.L2GenesisBlockBaseFeePerGas == nil {
 		return fmt.Errorf("%w: L2 genesis block base fee per gas cannot be nil", ErrInvalidDeployConfig)
 	}
-	if d.ColosseumChallengeTimeout == 0 {
-		return fmt.Errorf("%w: ColosseumChallengeTimeout cannot be 0", ErrInvalidDeployConfig)
+	if d.ColosseumBisectionTimeout == 0 {
```

Comment:
Wouldn't a comparison with the same values as the defaults set in deploy-config be meaningless?

Reply:
AFAIK, default value in deploy-config doesn't affect golang json unmarshaller. So if the value is not set, the default value will be 0.